socketio_block.py

```diff
@@ -36,63 +28,65 @@ class SocketIO(Retry, Block):
         version (enum): Which version of socketIO to use
 
     """
-    version = VersionProperty('1.1.0')
+    version = VersionProperty('2.0.0')
     host = StringProperty(title='SocketIo Host', default="127.0.0.1")
     port = IntProperty(title='Port', default=443)
-    room = StringProperty(title='SocketIo Room', default="default")
+    room = StringProperty(title='Socket.io Room', default="default")
     content = ExpressionProperty(
-        title='Content', default="{{json.dumps($to_dict(), default=str)}}",
+        title='Content', default="{{ json.dumps($to_dict(), default=str) }}",
         visible=False)
     listen = BoolProperty(title="Listen to SocketIo Room", default=False)
-    socketio_version = SelectProperty(
-        SocketIOVersion, title='Socket.IO Version', default=SocketIOVersion.v1)
+    connect_timeout = TimeDeltaProperty(
```

Comment:
Isn't it desired to change this value through the UI, why is visible set to False?